src/index.html

```diff
@@ -3,14 +3,18 @@
 ---
 
 <header class="hero">
-  <h1 class="hero__title">
-    British Columbia Made
-    <small>Since 1987</small>
-  </h1>
-
-  <p class="hero__tagline">Premium Quality Industrial Safety Garments</p>
-
-  <a class="button button--accent" href="/products">Our Products</a>
+  <div class="title__container">
```

Comment:
`.hero__container` or `.hero_wrapper` is a more appropriate name for this.

Reply:
In the case where I need another container in the hero with different styling would I start appending numbers? E.g. hero__container_2?

Reply:
I think if you need more than 1 hero container, SOMEBODY did something wrong (a designer or a dev), and the underlying issue should be fixed.

Reply:
If you encounter a secondary container requirement, take a step back and think about what it is encapsulating. You'll most likely find that it will be a component as well as the other, so each should be encapsulated and named accordingly.

Reply:
Sorry I was thinking in terms of rows not containers, which is why I gave it a name specific to title. Definitely miss-named it though.
E.g.
```
<header class="hero">
  <div class="title__row">
      <h1 class="hero__title">
         British Columbia Made
         <br>
         <small>Since 1987</small>
      </h1>
      <p class="hero__tagline">Premium Quality Industrial Safety Garments</p>
      <div class="hero__button__row">
          <a class="button button-yellow" href="/products">VIEW OUR PRODUCTS</a>
      </div> 
  </div>
  <div class="diagram__row">
   // Stuff in here
  </div>
</header>